### PR TITLE
Ana/4773 polkadot ux improvement: Select and Deselect on PageValidator

### PR DESCRIPTION
**Pull request by Bitcoinera**
Related to #4773 

**Description:**

- [x] Add stake button to TableBalances BalancesRow
- [x] Hide Stake/Unstake on PageValidator
- [ ] Add Select/Deselect on PageValidator

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f4d0889e2799f40f716a46e--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
Sadly failing on "Select", although debugging it seems all good at first sight

image

**Comment by Bitcoinera** [comment]
What is happening with the E2E? This is the reason why this is still a WIP

File: app/tests/unit/specs/components/staking/__snapshots__/PageValidator.spec.js.snap [hunk 1]
```diff
@@ -109,7 +109,7 @@ exports[`PageValidator if uptimePercentage is blank 1`] = `
       customclass=""
       id="undelegation-btn"
       type="secondary"
-      value="Unstake"
+      value="Stake"
```
Comment (faboweb) [inline app/tests/unit/specs/components/staking/__snapshots__/PageValidator.spec.js.snap:112]: this looks wrong

Comment (Bitcoinera) [reply app/tests/unit/specs/components/staking/__snapshots__/PageValidator.spec.js.snap:112]: Yeah, weird that it changed... Good eye

Comment (Bitcoinera) [reply app/tests/unit/specs/components/staking/__snapshots__/PageValidator.spec.js.snap]: Fixed

**Comment by Bitcoinera** [comment]
Now I see why Polkadot E2E are failing here :eyes: 

image

**Comment by Bitcoinera** [comment]
ah damn, E2E failing because of API change.... :facepalm:

**Comment by Bitcoinera** [comment]
Ready! :woman_dancing:

**Comment by Bitcoinera** [comment]
OK, now I see why E2E tests are failing here :eyes: 

```
  NoSuchElementError: An error occurred while running .setValue() command on <#amount>: undefined; undefined
   {"sessionId":"a4d612771446f99bfb4ff52a3cd58144354f40f5","state":"success","value":[],"status":0}
```

**Comment by Bitcoinera** [comment]
Let's merge! The Polkadot E2E are failing due to this error:

image

And this error is just due to some kind of websockets/rpc connection error

**Comment by Bitcoinera** [comment]
Another observation. This PR also includes #4787 actually. So it is better to merge #4787 first and then this one. Makes more sense

**Comment by Bitcoinera** [comment]
I am a believer :musical_note: :notes:

**Pull request merged**